rosa/rosa-wrapper.py

```diff
@@ -213,16 +231,12 @@ def _build_cluster(rosa_cmnd,cluster_name_seed,expiration,rosa_azs,my_path,es,in
         with open(cluster_path + "/" + 'installation.log',"r") as output_file:
             logging.error(output_file.read())
         success = False
-        # Only extending expiration on failed clusters, because ready one will be deleted at the end of the script
-        # if expiration:
-        #     logging.info('Extending cluster expiration on %d minutes' % expiration)
-        # rosa edit cluster -c 1iaaehmdd23lhifqk4fsjghrci82nt51 --expiration-time=2021-01-22T03:05:42.44677Z
-        # rosa edit cluster -c 1iaaehmdd23lhifqk4fsjghrci82nt51 --expiration=72h
-        # https://issues.redhat.com/browse/SDA-3600
     else:
         kubeconfig_path = _download_kubeconfig(metadata['cluster_id'], cluster_path)
         _add_machinepool(rosa_cmnd,kubeconfig_path,metadata['cluster_id']) if args.machinepool_name else None
         _install_addons(rosa_cmnd,metadata['cluster_id'], addons) if addons else None
+    # extending cluster expiration on all clusters
+    _expiration(rosa_cmnd,metadata['cluster_id'], expiration) if expiration else None
```

Comment:
What is the purpose of the ternary condition (if/else) here?

Reply:
expiration function will be triggered only if expiration parameter is defined.
Inline conditionals in python requires the complete definition of the conditional, else option is mandatory